synapse/storage/schema/main/delta/68/06as_txn_seq.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2022 Beeper
```

Comment:
it would be good to bump `synapse.storage.schema.SCHEMA_VERSION` (and move this to `69/`). Then in future, when we remove the upsert on `application_services_state`, we can bump `SCHEMA_COMPAT_VERSION` to ensure that nobody tries rolling back to a version of synapse that relies on it.

Reply:
https://github.com/matrix-org/synapse/pull/12209/commits/d62adeee3e503b0094b62f21cf20e141687138fb